src/Treeview.ts

```diff
@@ -302,6 +302,11 @@ export class JsonOutlineProvider implements vscode.TreeDataProvider<string> {
             if (keyNode instanceof Json.StringValue) {
                 return keyNode.unquotedValue;
             }
+        } else {
```

Comment:
This can be simplified/partially combined

Reply:
Fixed.